src/styles/profile.css

```diff
@@ -2,7 +2,6 @@
   padding: 8px;
   display: flex;
   align-items: flex-start;
```

Comment:
Default is start then why explicitly mention

Reply:
default value is stretch for align items